discretisedfield/field.py

```diff
@@ -3520,7 +3520,7 @@ def rfftn(self, **kwargs):
         axes = range(self.mesh.region.ndim)
         ft = spfft.fftshift(
             spfft.rfftn(self.array, axes=axes, **kwargs),
-            axes=axes,
+            axes=axes[:-1],
         )
```

Comment:
Why is there no `fftshift` in the mesh? https://github.com/ubermag/discretisedfield/blob/053de0c2b94c17d00458b4bc31bb3c0edf61af35/discretisedfield/mesh.py#L2149

Edit: we only use `fftfreq` to find `f_min` and `f_max`. It might be useful to add a comment to `mesh.fftn` stating why `fftshift` is not required.